app/models/chouette/route.rb

```diff
@@ -7,7 +7,7 @@ class Route < Chouette::TridentActiveRecord
     include ObjectidSupport
     extend Enumerize
 
-    if ENV["CHOUETTE_ROUTE_POSITION_CHECK"] == "true" || !Rails.env.production?
+    if ChouetteEnv["CHOUETTE_ROUTE_POSITION_CHECK"] || !Rails.env.production?
```

Comment:
ici c'est plutôt un boolean, isn't it ?

Reply:
Oui mais je l'ai déclaré comme booléen à la base, donc il sera casté en booléen quoi qu'il arrive